scanner/src/main/kotlin/Main.kt

```diff
@@ -250,10 +250,15 @@ object Main {
                 packages.addAll(analyzerResult.packages)
             }
 
-            packages.forEach { pkg ->
-                val entry = pkgSummary.getOrPut(pkg.id.toString()) { SummaryEntry() }
-                entry.scopes.addAll(findScopesForPackage(pkg, analyzerResult.project))
-                scanEntry(entry, pkg.id.toString(), pkg)
+            val results = scanner.scan(packages, outputDir, downloadDir)
+            results.forEach {
+                pkgSummary.getOrPut(it.key.id.toString()) {
```

Comment:
I don't think we need `getOrPut()` here anymore as we're not using the return value anyway... can't we just map `results` to `pkgSummary`?